### DISCUSSION TITLE
article/notion-next-ackee

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#91] [Announcements]
# article/notion-next-ackee

这是一个由NotionNext生成的站点

https://docs.tangly1024.com/article/notion-next-ackee

**Comment by qixing-jk** [comment]
配置NotionNext里的NEXT_PUBLIC_ANALYTICS_ACKEE_TRAKER写错了，少了一个C，应为TRACKER，让我多搞了一个小时，呜呜呜，最后代码查询用法的时候才发现是写错了

**Reply by best** [reply]
感谢分享，让我快速定位到了问题，我完全没想到会是文档把环境变量名字写错了OvO
确实，正确的变量是 `NEXT_PUBLIC_ANALYTICS_ACKEE_TRACKER`，如果配置失败会没有加载用于统计的 JS
